hugo/content/engine.md

```diff
@@ -4,4 +4,6 @@ section: engine
 url: engine
 title: source{d} Engine
 socialDescription: Powerful language-agnostic analysis of your source code and git history.
+socialImage: img/engine-social-card.png
```

Comment:
I'd create new logos image into [`static/img/logos`](https://github.com/src-d/landing/tree/master/static/img/logos), so they'll be all in the same dir.
```suggestion
socialImage: img/logos/engine-social-card.png
```